packages/portal-proto/src/features/cohortComparison/SurvivalCard.tsx

```diff
@@ -126,6 +126,8 @@ const SurvivalCard: React.FC<SurvivalCardProps> = ({
   ) => {
     return await createSet({
       filters: makeSurvivalCaseFilters(primarySetId, comparisonSetId),
+      intent: "user",
+      set_type: "frozen",
```

Comment:
<img width="852" alt="Screenshot 2024-04-05 at 2 37 02 PM" src="https://github.com/NCI-GDC/gdc-frontend-framework/assets/4624053/b0e8e65e-c5c7-4056-93f6-28654910a314">

Reply:
"portal" (did you hear there is a new portal game?)